spacy/training/pretrain.py

```diff
@@ -60,10 +60,14 @@ def pretrain(
     row_settings = {"widths": (3, 10, 10, 6, 4), "aligns": ("r", "r", "r", "r", "r")}
     msg.row(("#", "# Words", "Total Loss", "Loss", "w/s"), **row_settings)
 
-    def _save_model(epoch, is_temp=False):
+    def _save_model(epoch, is_temp=False, is_last=False):
         is_temp_str = ".temp" if is_temp else ""
         with model.use_params(optimizer.averages):
-            with (output_dir / f"model{epoch}{is_temp_str}.bin").open("wb") as file_:
+            if is_last:
+                save_path = output_dir / f"model_last.bin"
```

Comment:
I think it would be nicer if the name is more similar to `spacy train`, but I don't have really strong feelings about it if there's a good reason to keep `model_last`?

```suggestion
                save_path = output_dir / f"model-last.bin"
```

Reply:
Oh I didn't realize it was a "-"! I agree; let's change it